web/themes/custom/liibarrio2020/css/custom.css

```diff
@@ -0,0 +1,9 @@
+/*Judgment Listing View*/
+
+.view--liibarrio .view-content:not(:last-child){
+border-bottom: 1px solid #d5d5d5 !important;
```

Comment:
Please use one of the Bootstrap theme color variables, such as $border-color or whatever is appropriate here.

This seems very broad and not specific to the judgment listing -- is it correct? Can you include a screenshot?